central/networkgraph/aggregator/aggregator_impl.go

```diff
@@ -225,6 +225,45 @@ func (a *aggregateExternalConnByNameImpl) Aggregate(flows []*storage.NetworkFlow
 	return ret
 }
 
+type aggregateAllImpl struct{}
+
+// Aggregate aggregates deduplicates all connections into a single connection.
+func (a *aggregateAllImpl) Aggregate(flows []*storage.NetworkFlow) []*storage.NetworkFlow {
+	normalizedConns := make(map[networkgraph.NetworkConnIndicator]*storage.NetworkFlow)
+	ret := make([]*storage.NetworkFlow, 0, len(flows))
+
+	for _, flow := range flows {
+		srcEntity, dstEntity := flow.GetProps().GetSrcEntity(), flow.GetProps().GetDstEntity()
+		// This is essentially an invalid connection.
+		if srcEntity == nil || dstEntity == nil {
+			utils.Should(errors.Errorf("network conn %s without endpoints is unexpected", networkgraph.GetNetworkConnIndicator(flow).String()))
+			continue
+		}
+
+		flow = flow.CloneVT()
+
+		flowProps := flow.GetProps()
+		if flowProps == nil {
+			continue
+		}
```

Comment:
Having a separate variable implies that we need to use it later, so might be a little clearer to do it all in the if: 

```suggestion
		if flow.GetProps() == nil {
			continue
		}
```